src/hooks/constants/lib/getRepayFee.test.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiPromise } from '@polkadot/api';
```

Comment:
needs improvement

Reply:
test as simple function;  no need to reproduce the apollo infrastructure

Reply:
test implementation,  not mock.
use https://stackoverflow.com/a/43744255 with createtype to use the implementation code